ncl/ncl_index/ncl-index-table.csv

```diff
@@ -52,3 +52,15 @@ NCL Function,Description,Python Equivalent,Notes
 `satvpr_slope_fao56 <https://www.ncl.ucar.edu/Document/Functions/Crop/satvpr_slope_fao56.shtml>`__," Compute the slope of the saturation vapor pressure curve as described in FAO 56","``geocat.comp.saturation_vapor_pressure_slope()``",`example notebook <../ncl_entries/meteorology.ipynb#satvpr-slope-fao56>`__
 `coriolis_param <https://www.ncl.ucar.edu/Document/Functions/Contributed/coriolis_param.shtml>`__,"Calculate the Coriolis parameter","``metpy.calc.coriolis_parameter()``",`example notebook <../ncl_entries/meteorology.ipynb#coriolis-param>`__
 `csc2s <https://www.ncl.ucar.edu/Document/Functions/Built-in/csc2s.shtml>`__,"Converts Cartesian coordinates on a unit sphere to spherical coordinates (lat/lon)","``astropy.coordinates.representation``",`example notebook <../ncl_entries/great_circle.ipynb#csc2s>`__
+`boxplot <https://www.ncl.ucar.edu/Document/Functions/Shea_util/boxplot.shtml>`__,"Creates a boxplot","``matplotlib.axes.Axes.boxplot`` and ``matplotlib.pyplot.boxplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Boxplots/index.html>`__
```

Comment:
```suggestion
`boxplot <https://www.ncl.ucar.edu/Document/Functions/Shea_util/boxplot.shtml>`__,"Creates a boxplot","``matplotlib.pyplot.boxplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Boxplots/index.html>`__
```

I know I suggested this, but rather than changing this and everything below to have both the pyplot wrapper and the Axes API - maybe just leaving it as only referencing the pyplot is fine.